File: src/main/java/de/tuberlin/dima/model/Person.java
package de.tuberlin.dima.model;

import java.io.Serializable;
import java.util.*;

public class Person implements Serializable{

    private String name;
    private String school;
    private char sex;
    private int age;
    private String major = "";
    private List<String> bestCourse = new ArrayList<String>();
    private List<String> jobs = new ArrayList<String>();

    public String getMajor() {
        return major;
    }

    public void setMajor(String major) {
        this.major = major;
    }

    public List<String> getBestCourse() {
        return bestCourse;
    }

    public void setBestCourse(List<String> bestCourse) {
        this.bestCourse = bestCourse;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSchool() {
        return school;
    }

    public void setSchool(String school) {
        this.school = school;
    }

    public char getSex() {
        return sex;
    }

    public void setSex(char sex) {
        this.sex = sex;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public List<String> getJobs() {
        return jobs;
    }

    public void setJobs(List<String> jobs) {
        this.jobs = jobs;
    }

    @Override
    public String toString() {
        StringBuffer person = new StringBuffer().append(name).append(";")
                .append(school).append(";").append(sex).append(";")
                .append(age).append(";").append(major).append(";");

        int counter = 0;
        for(String course : bestCourse){
            person.append(course);
            if(counter < bestCourse.size()-1) {
                person.append(";");
                counter++;
            }
        }

        counter = 0;
        for(String job : jobs){
            person.append(job);
            if(counter < jobs.size()-1) {
                person.append(";");
                counter++;
            }
        }
        return person.toString();
    }
}

File: src/main/scala/de/tuberlin/dima/model/Person.scala
package de.tuberlin.dima.model

import scala.collection.mutable.ArrayBuffer

class Person{

	private var _name: String = _
	private var _school: String = _
	private var _sex: Char = _
	private var _age: Int = _
	private var _major: String = _

	// ArrayBuffer is Mutable by itself
	private val _courses: ArrayBuffer[String] = new ArrayBuffer[String]
	private val _jobs: ArrayBuffer[String] = new ArrayBuffer[String]

	def name = _name
	def name(s:String):Unit = {_name = s}

	def school = _school
	def school(s:String):Unit = {_school = s}

	def sex = _sex
	def sex(c:Char):Unit = {_sex = c}

	def age = _age
	def age(value:Int):Unit = {_age = value}

	def major = _major
	def major(s:String):Unit = {_major = s}

	def courses = _courses
	def courses(arrayBuffer: ArrayBuffer[String]): Unit = {
		_courses ++=(arrayBuffer)
	}

	def jobs = _jobs
	def jobs(arrayBuffer: ArrayBuffer[String]): Unit = {
		_jobs ++=(arrayBuffer)
	}

	override def toString: String = {
		_name + " " + _school + " " + _sex + " " + _age + " " + _major + "[" + _courses.mkString(";") + "]" + "	" + "[" + _jobs.mkString(";") + "]"
	}

}